.github/workflows/ci.yml

```diff
@@ -45,6 +39,13 @@ jobs:
         ./configure
         make
 
+    - name: Install dependencies
+      run: |
+        php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+        php composer-setup.php
+        php -c php.ini composer.phar require --dev --ignore-platform-reqs phpunit/phpunit "4.8.36"
```

Comment:
This step is a weird hack, right?  please explain why we're manually installing this particular version of PHPUnit, instead of getting it automatically via dependencies

My policy is: any time you need to do a weird hack to make something work, leave a comment so the next person understands why it's necessary. This also let's them know when they could remove a hack some day because "oh, we don't need that old version anymore so we can remove this line". When there's no comment, the hack stays in superstitiously because the new person doesn't really understand why it's there

Reply:
actually we don't need to do the hack, I removed if we add the dependency of old version of phpunit